psi4/driver/qcdb/util/mpl.py

```diff
@@ -0,0 +1,34 @@
+
+def plot_coord(ref, cand=None, orig=None, comment=None):
+    """Display target geometry `ref` as black dots in 3D plot. If present, also
+    plot candidate geometry `cand` as red dots and starting geometry `orig` as
+    pale blue dots. Plot has text `comment`. For assessing alignment, red and
+    black should overlap and pale blue shows where red started.
+    
+    """
+    import numpy as np
+    from matplotlib import pyplot
```

Comment:
I'm guessing this isn't called by everyday users, but it may still be worth a `try / except ImportError` for the not entirely ubiquitous matplotlib.

Reply:
Hopefully they won't have `do_plot=True` w/o considering if they have anything to plot with, but yes, good point, there's several feature that require add'l packages that I've try/excepted so good form to do mpl, too.

Reply:
I've tripped myself up a time or two by assuming that mpl is installed on a given machine, before discovering it isn't.  I'm not aware of any distros that ship it by default so I think a message describing how to pull it in through conda may aid novice users who thought they'd just get a plot automagically.  Other stuff that's more standard is fine to leave unguarded IMHO